app/src/main/res/layout/top_placeholder_item.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This could optionally just be a <View> with a self-closing "/>" (it could be more performant than a ConstraintLayout too)

`<View xmlns:android="http://schemas.android.com/apk/res/android"
android:layout_width="match_parent"
    android:layout_height="0dp"/>
`

Reply:
Good idea.  Not sure why I stayed with contraint layout.  I'll try it.  Thanks!

Reply:
Works well.  Updated.  Thanks!